### PR TITLE
add support for PXE UEFI boot

### PR DESCRIPTION
**Pull request by calvix**
for CG we need to add support for PXE UEFI boot, which needs different  pxe image and some extra options

### CODE REVIEW

File: pxemgr/ipxe_handlers.go [hunk 1]
```diff
@@ -36,7 +36,12 @@ func (mgr *pxeManagerT) ipxeBootScript(w http.ResponseWriter, r *http.Request) {
 
 	buffer := bytes.NewBufferString("")
 	buffer.WriteString("#!ipxe\n")
-	buffer.WriteString(fmt.Sprintf("kernel %s/images/vmlinuz coreos.autologin maybe-install-coreos=stable console=ttyS0,115200n8 mayu=%s next-script=%s\n", mgr.pxeURL(), mgr.pxeURL(), mgr.pxeURL()+"/first-stage-script/__SERIAL__"))
+	if mgr.config.Network.UEFI {
+		buffer.WriteString(fmt.Sprintf("kernel %s/images/vmlinuz coreos.autologin maybe-install-coreos=stable console=ttyS0,115200n8 mayu=%s next-script=%s\n initrd=initrd.cpio.gz", mgr.pxeURL(), mgr.pxeURL(), mgr.pxeURL()+"/first-stage-script/__SERIAL__"))
```
Comment (r7vme) [inline pxemgr/ipxe_handlers.go]: Difference between two lines is only `initrd=initrd.cpio.gz`.

1) Where it comes from? May be to put short comment?
2) Probably it should be `initrd=%s/initrd.cpio.gz"`?

Comment (calvix) [inline pxemgr/ipxe_handlers.go]: 1. its extra argument that needs to be add to kernel in order to boot with UEFI it comes from here - https://github.com/coreos/bugs/issues/1239#issuecomment-268000634
2. no, it should be only`initrd=initrd.cpio.gz`

Comment (r7vme) [inline pxemgr/ipxe_handlers.go]: looking thru [CoreOS official docs](https://github.com/coreos/docs/blob/master/os/booting-with-ipxe.md) mentioned there. It looks like they use initrd everywhere by default. Despite the fact that it was added for EFI specifically, it doesn't break legacy boot.

Do we really need additional UEFI optionin Mayu? Can we just add initrd by default?

Comment (calvix) [inline pxemgr/ipxe_handlers.go]: we could may add `initrd ` to all boots but  we need different PXE images for UEFI and Legacy, so we still need that option

**Pull request merged**